.storybook/dbTheme.js

```diff
@@ -0,0 +1,30 @@
+import { create } from '@storybook/theming/create';
+
+const TEXT = '#131821';
+const GREY = '#FDFDFD';
```

Comment:
maybe we could reference the existing tokens directly from `node_modules/@db-ui/base/build/js/tokens.js`

Reply:
good point, I'll try this